weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OverviewPage/ProvidersTab.tsx

```diff
@@ -0,0 +1,132 @@
+import {ApolloProvider} from '@apollo/client';
+import {GridColDef, GridRenderCellParams} from '@mui/x-data-grid-pro';
+import {makeGorillaApolloClient} from '@wandb/weave/apollo';
+import {useViewerInfo} from '@wandb/weave/common/hooks/useViewerInfo';
+import {Icon} from '@wandb/weave/components/Icon';
+import {Pill} from '@wandb/weave/components/Tag';
+import {Tailwind} from '@wandb/weave/components/Tailwind';
+import React from 'react';
+
+import {Link} from '../common/Links';
+import {useConfiguredProviders} from '../PlaygroundPage/useConfiguredProviders';
+import {WFDataModelAutoProvider} from '../wfReactInterface/context';
+import {ProviderTable} from './ProviderTable';
+
+const ProviderStatus = ({isActive}: {isActive: boolean}) => {
+  return (
+    <div className="flex items-center">
+      <Pill
+        color={isActive ? 'green' : 'moon'}
+        label={isActive ? 'Configured' : 'Not configured'}
+        icon={isActive ? 'checkmark-circle' : 'failed'}
+      />
+    </div>
+  );
+};
+
+const columns: GridColDef[] = [
+  {
+    field: 'name',
+    headerName: 'NAME',
+    flex: 0.2,
+    minWidth: 200,
+  },
+  {
+    field: 'status',
+    headerName: 'STATUS',
+    flex: 0.2,
+    minWidth: 200,
+    renderCell: (params: GridRenderCellParams) => (
+      <div className="flex h-full items-center">
+        <ProviderStatus isActive={params.value} />
+      </div>
+    ),
+  },
+  {
+    field: 'missingSecrets',
+    headerName: 'Missing Secrets',
+    flex: 0.4,
+    minWidth: 200,
+    renderCell: (params: GridRenderCellParams) => (
+      <div className="flex h-full items-center justify-between">
+        <span className="text-moon-500">{params.value || 'None'}</span>
+        {params.row.isAdmin && !params.row.status && (
+          <Link
+            to={`/${params.row.entityName}/settings`}
+            $variant="secondary"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="flex items-center gap-2">
+            {'Add team secret'}
+            <Icon name="forward-next" />
+          </Link>
+        )}
+      </div>
+    ),
+  },
+];
+
+export const ProvidersTabInner: React.FC<{
+  entityName?: string;
+  projectName?: string;
+}> = ({entityName = ''}) => {
+  const {loading: loadingUserInfo, userInfo} = useViewerInfo();
+  const isAdmin = !loadingUserInfo && userInfo?.admin;
```

Comment:
I think this logic might not be right - I think we need to check whether the user has the admin role on the project?